### ISSUE TITLE
post 0.9 balance request

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoIssues#15561]
(still about wood pulp, lol)
Discussed in discord with @mirasrael 

1. you can craft pitch from wood pulp or plant fibers
2. you can craft mortared stone (and maybe stone road) with "mortar" tag, contain mortar and pitch
3. add more realistic mortar recipes (filler+binder, e.g. sand+clay, sand+limestone, limestone(bigger amount), clay+[naturalfibers(tag)], sand+pitch, etc). Can be balanced by setting output quantity.
4. more advanced blocks (e.g. bricks) require mortar (not pitch).

**Comment by mirasrael** [comment]
Yeah, I like this idea. We can make more realistic Mortar, more interesting production chains and don't break early game in this case.

**Comment by elfl0rd** [comment]
And yes, with this we can have wood pulp from more carpentry productions.
Hewn logs, lumber and furniture maybe.

**Comment by trickenso** [comment]
Not wanting to complicate things unnecessarily, but perhaps there could be a dedicated wood pulp production chain, that directly processes logs or other timber products:
https://en.wikipedia.org/wiki/Pulp_(paper)#Preparation_for_pulping

Apparently in real life, wood must be debarked and then is often entirely reduced to chips and then pulp. It's unclear if you'd want chipped bark as a byproduct, but perhaps the processes could create chips that then need further processing to create pulp? Chips could behave like the wooden equivalent of crushed stone, and could have direct applications in things like mortar recipes or as fuel.
Chips could still be produced as a byproduct, but it seems reasonable for them to also have their own recipes considering it's a dedicated industry in reality.

Pulp production is also a major polluter of waterways, so it's a shame to overlook that opportunity:
https://en.wikipedia.org/wiki/Pulp_(paper)#Environmental_concerns
